### ISSUE TITLE
This API key is for API version 5. You tried using version 4

### ISSUE THREAD

**Issue by chengtie** [ArkeologeN/pipl#3]
Hello,

Thanks for making this wrapper.

I have tried this code in my backend:

```
router.post('/httpOnly/enrich/pipl', (req, res) => {
    var pipl = require('pipl')('BUSINESS-PREMIUM-DEMO-xyozhmgkc57085z7...');
    pipl.search.query('json', {"email": "chengtie@gmail.com"}, function(err, data) {
        console.log(data);
        res.json(data)
    });
})
```

And I got this error:

```
index.js POST /httpOnly/enrich/pipl
http://api.pipl.com/search/v4/?email=chengtie%40gmail.com&key=BUSINESS-PREMIUM-DEMO-xyozhmgkc57085z7q4xoqunq
{ '@http_status_code': 403,
  error: 'This API key is for API version 5. You tried using version 4' }
```

Do you know how to fix this?

Thank you

**Comment by DavideViolante** [comment]
The PR I just opened resolves the issue. I hope the owner will merge it!

**Comment by ArkeologeN** [comment]
The issue is already fixed in `0.1.2`, therefore, I'm closing this.
